iroh-sync/src/ranger.rs

```diff
@@ -293,53 +292,10 @@ pub trait Store<E: RangeEntry>: Sized {
         prefix: &E::Key,
         predicate: impl Fn(&E::Value) -> bool,
     ) -> Result<usize, Self::Error>;
-}
-
-#[derive(Debug)]
-pub struct Peer<E: RangeEntry, S: Store<E>> {
```

Comment:
Cool that is neat to make the methods just default impls on the `Store`, I think I like this.